plugin.audio.itvrs_radio_ice/default.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
+
+import sys
+import xbmcgui
+import xbmcplugin
+
+addon_handle = int(sys.argv[1])
+
+xbmcplugin.setContent(addon_handle, 'audio')
+
+url = 'http://stream.zeno.fm/3611cwn8mbruv'
+li = xbmcgui.ListItem(label='ITVRS - Radio Ice', iconImage='DefaultAudio.png')

```

Comment:
Please don't use `iconImage` it will break for Matrix. Use `listitem.setArt` only.